src/client.cpp

```diff
@@ -83,7 +85,7 @@ void set_client_time_out_timer(struct itimerval *timer, TicksTime testStart) {
     double sampleTimeSeconds = sampleTime.toDecimalUsec() / 1000000;    // secs
     double sampleSecObsRate = sampleTimeSeconds / sampleObservations;   // secs/obs
     double runTimeEstimate = sampleSecObsRate * totalObservations;      // (secs/obs)*obs = secs
-    double waitingCap = 1.5 * runTimeEstimate; // Add leeway to process stats and write to file
+    double waitingCap = 1.5 * runTimeEstimate + 10; // Add leeway to process stats and write to file
```

Comment:
Per offline discussion,10 seconds are added here to guarantee some time to compute stats, regardless of observation count. I will look into deactivating timer at end of run so stats afterwards don't have to worry about timer going off. Waiting cap should be for run, not stats computing.